src/NuGet.Core/NuGet.CommandLine.XPlat/Strings.resx

```diff
@@ -712,4 +712,10 @@ For more information, visit https://docs.nuget.org/docs/reference/command-line-r
   <data name="AddClientCertCommandDescription" xml:space="preserve">
     <value>Adds a client certificate configuration that matches the given package source name.</value>
   </data>
-</root>
+  <data name="AddPkg_PackagePrerelease" xml:space="preserve">
+    <value>Consider prerelease packages when determining the latest version to install.</value>
+  </data>
+  <data name="Error_PrereleaseWhenVersionSpecified" xml:space="preserve">
+    <value>Prerelease and Version options cannot be combined.</value>
```

Comment:
I think we should be explicit about the option names. 

Example: https://github.com/NuGet/NuGet.Client/blob/9f277be9d55cff0aeb23de18fd62d543a41ff10a/src/NuGet.Core/NuGet.CommandLine.XPlat/Strings.resx#L328